rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyPrinter.java

```diff
@@ -24,6 +24,7 @@
 import org.openrewrite.groovy.tree.GContainer;
 import org.openrewrite.groovy.tree.GRightPadded;
 import org.openrewrite.groovy.tree.GSpace;
+import org.openrewrite.internal.StringUtils;
```

Comment:
```suggestion
```

Reply:
The reminded me of [The Queen's Duck](https://bwiggs.com/notebook/queens-duck/); not sure if it was intended that way. ;)

Reply:
No, it was a remnant of older try-outs. Should be remove indeed.